src/components/10-atoms/carousel/README.md

```diff
@@ -1,21 +1,34 @@
 # Carousel
 
-Every child is rendered as a slide. Only one child is visible at one time.
+Each child is rendered as a slide. In default mode, only one child is visible at a time, and navigation arrows
+are provided. Also, auto-rotation to the right is enabled by default.
 
 ## Properties
 
 ### autorotatedisabled
 
-This Boolean attribute specifies if the slides will not automatically rotate.
+This Boolean attribute specifies whether slide auto-rotation should be suppressed (default: false).
 
 ### autorotatetime
 
-This number specifies the milliseconds the slides will automatically rotate.
+This numeric attribute specifies the waiting time in milliseconds for the next slide to appear in auto-rotation mode (default: 5000).
 
 ### keysenabled
 
-This boolean attribute specifies if the keyboard navigation (left/right keys) should be enabled.
+This Boolean attribute specifies whether keyboard navigation via left/right arrow keys should be enabled (default: false).
+
+### arrowstyle
+
+This string-valued attribute adds custom inline styling to the arrow buttons of the carousel
+(e.g. `border-color:grey` for grey arrows, or `display:none` to suppress arrows).
+
+### mode
+
+This string-valued attribute forces the carousel into a specific mode of behaviour (default: no specific mode of behaviour).
+
+Currently, one mode value is recognized: `all` switches to horizontal layout of child elements for viewports &gt;= 992px.
+In that layout, _all_ child elements are visible, they fill the viewport width seamlessly, and carousel navigation is suppressed.
 
 ## Child Elements
 
-Do not set more than ~100 child elements. Otherwise, because of height-calculation costs performance will suffer.
+There is an absolute limit of 10 child elements.
```

Comment:
Only 10? Why is this needed? As long as we don't have to "show all at once", there should not be a strict limit, just a recommendation, no?

This would be a massive breaking change, if we limit our users like that. They might have to add additional logic just to work with this limitation.